spark-publish/index.js

```diff
@@ -37,20 +38,24 @@ export const publish = async ({
   logger.log(`Publishing ${measurements.length} measurements`)
 
   // Share measurements
+  let start = new Date()
   const file = new File(
     [JSON.stringify(measurements)],
     'measurements.json',
     { type: 'application/json' }
   )
   const cid = await web3Storage.put([file])
+  const uploadMeasurementsDuration = new Date() - start
   logger.log(`Measurements packaged in ${cid}`)
 
   // Call contract with CID
   logger.log('ie.addMeasurements()...')
+  start = new Date()
   const tx = await ieContract.addMeasurements(cid.toString())
   const receipt = await tx.wait()
   const event = receipt.events.find(e => e.event === 'MeasurementsAdded')
   const { roundIndex } = event.args
```

Comment:
Nice! What do you think about updating `measurements` with the `roundIndex` value we got from the smart-contract interaction? That would solve the problem we discussed on Slack, where we end up with a part of measurements belonging to the previous round because of a race condition/timing issues.

Reply:
I mean writing `roundIndex` to Postgres DB.

Reply:
If we need that association, here seems to be a good place to do it 👍